app/controllers/samples_controller.rb

```diff
@@ -1426,4 +1428,19 @@ def check_owner
       # Rendering halts the filter chain
     end
   end
+
+  def warn_if_large_bulk_upload(samples)
+    # 100 is the limit that was chosen during covid19 surge for biohub projects.
+    if samples.length < 100
+      return
+    end
+    project = samples.first.project
+    uploader = samples.first.user
+    msg = "[Datadog] LargeBulkUploadEvent: #{samples.length} samples by #{uploader.role_name} in project #{project.name}." \
```

Comment:
Lets not put project name here. The status should be enough.

Reply:
For privacy? The project name I feel reveals a lot of information about the intent and priority of the uploads.

Reply:
Yes, for privacy. That field is not controlled by use, so it should be exposed to logs. 
Does not matter how useful it might be and more info is just a click away

Reply:
Ok I'll remove it. FWIW, we have been logging the project name elsewhere.

Reply:
OK, I'll approve a PR that changes that :P